tests/acceptance/controllers/register-test.js

```diff
@@ -0,0 +1,68 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { visit, fillIn, click } from '@ember/test-helpers';
+
+module('Acceptance | Controllers | register', function(hooks) {
```

Comment:
I think it should either be an acceptance-test or a controller-test, don't mix the two.